designs/developer/search-45/index.md

```diff
@@ -2,6 +2,43 @@
 parent: Developer
 version: 4.5
 ---
-# Search
+# Search - Add to Navigation in Dev Perspective
 
-Coming soon.
+## Overview
+This feature is enabled on the Search page in the Developer perspective, and is not available in the Administrator perspective.
+
+* On the Search page, users will be able add and remove resources to the navigation menu in the Developer perspective.
+* Added resources will be saved per user per machine.
+* If there are resources added, a separator will be added below Search (the last navigation item) in the Dev Perspective navigation.  
+* Added items will be displayed as primary nav items under the separator, in the order they were added.
+* The user should be able to unpin directly from the navigation.
+Config Maps & Secrets will be pre-pinned by default, but users can remove these items.
+* There is no limit to the number of items the user can add to the navigation.
+
+## Adding to navigation -from Search page
+* On the Search page, a “Add to navigation” button link is available on the accordion title.
+* Clicking on the Add to navigation button will add the Resource Type to the left navigation, under the separator.
+* The Add to navigation link then changes to Remove from navigation, as shown in the next screen.
+
+![search-01](img/search-01.png)
+Figure 1 - Add to navigation
+
+## Removing from navigation - from Search page
+On the Search page, if a resource type is already added to nav, the button link label is changed to Remove from navigation.
+
+![search-02](img/search-02.png)
+Figure 2 - Remove from navigation
+
+## Removing from navigation - from navigation
+![search-03](img/search-03.png)
+Hovering over an item on the navigation menu will display an selectable icon.  Clicking on this icon allows the user to remove the item from the Nvigation menu.
+Figure 3 - Hover over nav item
+
+## Confirmation
+A confirmation dialog is displayed when removing an item from navigation.
+
+![search-04](img/search-04.png)
+Figure 4 - Confirmation  
+
+## Mobile Considerations
+In mobile, the Add to/Remove from navigation button link should be stacked under the resource name in the accordion.
```

Comment:
Actually, i just tested it, they didn't end up having to stack. Unsure what you mean about hover ... this is a button link, but I may be missing what you're saying!
<img width="505" alt="Screen Shot 2020-05-17 at 11 45 14 AM" src="https://user-images.githubusercontent.com/12504403/82153178-f2a6e780-9833-11ea-829b-5780c6086ba4.png">


I'll pull this section out

Reply:
On mobile, you can remove them from the link button on the Search page OR you can remove them from the nav menu.  The only difference is, you don't see the tooltip when hovering over the icon when on a touch device!

Reply:
Ah okay, so you always see the remove icon on mobile?

Reply:
Looks like not:

![image](https://user-images.githubusercontent.com/9122899/82239252-7df0ad80-9906-11ea-9d96-fbd72e227da7.png)

But while holding your finger down on the nav item (which approximates a hover state) the icon does appear until you remove your finger again:

![image](https://user-images.githubusercontent.com/9122899/82239346-a24c8a00-9906-11ea-95ab-5bb9436a7189.png)

Tapping the invisible icon target does activate the "Remove from navigation?" modal though, so it is technically possible to remove the item from the nav.

I think this might be fixable using [CSS' interaction media query](https://caniuse.com/#feat=css-media-interaction) to always show the icon on touch-first devices. Maybe it should be grey rather than blue in that state though. Do we agree that'd be better, and we could file a bug?